tendrl/node_agent/alert/utils.py

```diff
@@ -0,0 +1,163 @@
+            if item.dir is True:
+                result[item.key.split("/")[-1]] = read(item.key)
+            else:
+                result[item.key.split("/")[-1]] = item.value
+    return result
+
+
+def get_alerts(alert):
+    alerts_arr = []
+    try:
+        if alert.classification == constants.NODE_ALERT:
+            alerts = read(
+                '/alerting/nodes/%s' % alert.node_id
+            )
+        elif  alert.classification == constants.CLUSTER_ALERT:
+            alerts = read(
+                '/alerting/clusters/%s' % alert.tags["integration_id"]
+            ) 
+    except EtcdKeyNotFound:
+        return alerts_arr
+    except (EtcdException, AttributeError) as ex:
+        raise ex
+    for alert_id, alert in alerts.iteritems():
+        if alert:
+            alerts_arr.append(AlertUtils().to_obj(alert))
+    return alerts_arr
+
+
+def classify_alert(alert, ttl=None):
+    if alert.classification == constants.NODE_ALERT:
+        NodeAlert(
+            alert_id=alert.alert_id,
+            node_id=alert.node_id,
+            time_stamp=alert.time_stamp,
+            resource=alert.resource,
+            current_value=alert.current_value,
+            tags=alert.tags,
+            alert_type=alert.alert_type,
+            severity=alert.severity,
+            classification=alert.classification,
+            significance=alert.significance,
+            ackedby=alert.ackedby,
+            acked=alert.acked,
+            ack_comment=alert.ack_comment,
+            acked_at = alert.acked_at,
+            pid=alert.pid,
+            source=alert.source,
+            delivered=alert.delivered
+        ).save(ttl=ttl)
+    elif alert.classification == constants.CLUSTER_ALERT:
+        ClusterAlert(
+            alert_id=alert.alert_id,
+            node_id=alert.node_id,
+            time_stamp=alert.time_stamp,
+            resource=alert.resource,
+            current_value=alert.current_value,
+            tags=alert.tags,
+            alert_type=alert.alert_type,
+            severity=alert.severity,
+            classification=alert.classification,
+            significance=alert.significance,
+            ackedby=alert.ackedby,
+            acked=alert.acked,
+            ack_comment=alert.ack_comment,
+            acked_at = alert.acked_at,
+            pid=alert.pid,
+            source=alert.source,
+            delivered=alert.delivered
+        ).save(ttl=ttl)
+
+
+def update_alert_count(alert, existing_alert):
+    alert_count_obj = []
+    if alert.classification == constants.NODE_ALERT:
+        counter_obj = NodeAlertCounters(
+            node_id=alert.node_id).load()
+        alert_count_obj.append(counter_obj) 
+    elif alert.classification == constants.CLUSTER_ALERT:
+        counter_obj = ClusterAlertCounters(
+            integration_id=alert.tags['integration_id']
+        ).load()
+        alert_count_obj.append(counter_obj)
+        if alert.resource == constants.VOLUME_ALERT:
```

Comment:
remove the VOLUME_ALERT check and move it o integrations.gluster.alerts.update_alert_count